### PR TITLE
4.8 Docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2995,6 +2995,30 @@ This setting determines whether team leave/join system messages are posted in th
...
+| This feature's ``config.json`` setting is ``"AllowCookiesForSubdomains": true`` with options ``true`` and ``false`` for above settings respectively.                 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Websocket URL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting allows the server to instruct clients where they should try to connect by websockets to.
+
+**True** XXXX Christopher
+
+**False** XXXX Christopher
```
Comment (amyblais) [inline source/administration/config-settings.rst]: Need help from Christopher on both settings.

Comment (jasonblais) [reply source/administration/config-settings.rst]: This is a URL, like the Site URL https://docs.mattermost.com/administration/config-settings.html#site-url

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2995,6 +2995,30 @@ This setting determines whether team leave/join system messages are posted in th
 | This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.     |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow Cookies for Subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting set's the Domain parameter on our cookies, which allows the browser to send the cookies to subdomains as well.
+
+**True** XXXX Christopher
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: True if cookies are allowed, false if not

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -137,7 +137,7 @@ Allow use of API v3 endpoints
 Set to false to disable all version 3 endpoints of the REST API. Integrations that rely on API v3 will fail and can then be identified for migration to API v4. API v3 is deprecated and will be removed in the near future. See https://api.mattermost.com for details.
 
 +---------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"EnableAPIv3": true`` with options ``true`` and ``false``.  |
+| This feature's ``config.json`` setting is ``"EnableAPIv3": false`` with options ``false`` and ``true``.  |
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Not surrounded in the box, so it won't render on the page.

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -2995,6 +2997,30 @@ This setting determines whether team leave/join system messages are posted in th
 | This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.     |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow Cookies for Subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting set's the Domain parameter on our cookies, which allows the browser to send the cookies to subdomains as well.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: To simplify, maybe we write this similarly to https://docs.mattermost.com/administration/config-settings.html#allow-banner-dismissal

I.e. No intro, just what happens when the setting is true or false.

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -2995,6 +2997,30 @@ This setting determines whether team leave/join system messages are posted in th
...
+
+**True** True when cookies are allowed.
+
+**False** False when cookies are not allowed.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"AllowCookiesForSubdomains": true`` with options ``true`` and ``false`` for above settings respectively.                 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Websocket URL
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: This should be written similar to the Site URL field (it's actually a string input, not a true/false field). https://docs.mattermost.com/administration/config-settings.html#site-url

Comment (lindy65) [reply source/administration/config-settings.rst]: Should "Subdomains" be hyphenated (i.e. "Sub-domains")?

Comment (amyblais) [reply source/administration/config-settings.rst]: @lindy65 In this case no, the config setting is written without the hyphen (based on [default.json file](https://github.com/mattermost/mattermost-server/blob/master/config/default.json)).

**Pull request merged**